diff --git a/tests/featuretests/profiler/utils/csv_parser.h b/tests/featuretests/profiler/utils/csv_parser.h
index 65ee441..d3e3397 100644
--- a/tests/featuretests/profiler/utils/csv_parser.h
+++ b/tests/featuretests/profiler/utils/csv_parser.h
@@ -31,6 +31,7 @@ THE SOFTWARE.
 #include <map>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 /* --------------------------------------------------------------------------*/
 /**
diff --git a/tests/featuretests/profiler/discretetests/binary/multithreaded_test.cpp b/tests/featuretests/profiler/discretetests/binary/multithreaded_test.cpp
index 49c2612..63bf947 100644
--- a/tests/featuretests/profiler/discretetests/binary/multithreaded_test.cpp
+++ b/tests/featuretests/profiler/discretetests/binary/multithreaded_test.cpp
@@ -37,6 +37,7 @@ THE SOFTWARE.
 #include <memory>
 #include <stdexcept>
 #include <string>
+#include <sstream>
 #include <vector>
 
 #include "utils/csv_parser.h"
