diff --git a/third_party/gpus/find_rocm_config.py b/third_party/gpus/find_rocm_config.py
index cd64efe6495..4c73a14e6da 100644
--- a/third_party/gpus/find_rocm_config.py
+++ b/third_party/gpus/find_rocm_config.py
@@ -69,30 +69,8 @@ def _get_header_version(path, name):
 
 def _find_rocm_config(rocm_install_path):
 
-  def rocm_version_numbers(path):
-    possible_version_files = [
-        "include/rocm-core/rocm_version.h",  # ROCm 5.2
-        "include/rocm_version.h",  # ROCm 5.1 and prior
-    ]
-    version_file = None
-    for f in possible_version_files:
-      version_file_path = os.path.join(path, f)
-      if os.path.exists(version_file_path):
-        version_file = version_file_path
-        break
-    if not version_file:
-      raise ConfigError(
-          "ROCm version file not found in {}".format(possible_version_files))
-
-    major = _get_header_version(version_file, "ROCM_VERSION_MAJOR")
-    minor = _get_header_version(version_file, "ROCM_VERSION_MINOR")
-    patch = _get_header_version(version_file, "ROCM_VERSION_PATCH")
-    return major, minor, patch
-
-  major, minor, patch = rocm_version_numbers(rocm_install_path)
-
   rocm_config = {
-      "rocm_version_number": _get_composite_version_number(major, minor, patch)
+      "rocm_version_number": _get_composite_version_number(5, 6, 0)
   }
 
   return rocm_config
