diff --git a/library/CMakeLists.txt b/library/CMakeLists.txt
index e797694..8597352 100644
--- a/library/CMakeLists.txt
+++ b/library/CMakeLists.txt
@@ -79,7 +79,7 @@ target_include_directories( hipfft
   $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include/hipfft>
   $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
   $<BUILD_INTERFACE:${HIP_INCLUDE_DIRS}>
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/hipFFT>
   )
 
 if( BUILD_WITH_LIB STREQUAL "CUDA" )
